app.py

```
@@ -0,0 +1,136 @@
+            shutil.rmtree(db_dir)
+
+        vectorstore = Chroma.from_documents(
+            split_docs, embedding_instance, persist_directory=db_dir
+        )
+        vectorstore.persist()
+        logging.info(f"[문서 노드] Vector DB 저장 완료: {db_dir}")
+
+        os.remove(tmp_path)
+        return {"status": 200}
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)